src/firefly/js/tables/TableUtil.js

```diff
@@ -101,54 +200,64 @@ export function isTblDataAvail(startIdx, endIdx, tableModel) {
 }
 
 
-export function findTblById(id) {
+export function getTblById(id) {
     return get(flux.getState(),[TblCntlr.TABLE_SPACE_PATH, 'data', id]);
 }
 
 /**
- * return the table results area
+ * return the group information
+ * @param {string} tbl_group    the group to look for
  * @returns {*}
```

Comment:
you are right.  but, it's the default intelliJ produces.  so, it's everywhere.  i will slowly update the docs as i come across them.